src/main/java/org/littil/api/ApplicationLifeCycle.java

```diff
@@ -41,65 +47,81 @@ public class ApplicationLifeCycle {
     String authorizationsClaimName;
 
     @Inject
-    DataSource dataSource;
+    UserService userService;
+    @Inject
+    SchoolService schoolService;
+    @Inject
+    GuestTeacherService guestTeacherService;
 
     @Inject
     ManagementAPI managementAPI;
 
-    void onStart(@Observes StartupEvent ev) throws Auth0Exception {
-        log.info("The application is starting with profile " + ProfileManager.getActiveProfile());
-
-        if (insertDevData) {
+    void onStart(@Observes StartupEvent ev) {
+        log.info("The application is starting with profile {}",ProfileManager.getActiveProfile());
+        if (this.insertDevData) {
             persistDevData();
-            cleanLittilAuth0User();
         }
     }
 
-    //todo fix this. This won't work when developing simultaneously
-    private void cleanLittilAuth0User() throws Auth0Exception {
-        log.info("Removing all authorizations from Auth0 appMetadata, this might cause issues when 2 developers are testing at once.");
+    void persistDevData() {
+        log.info("Persisting auth0 user data to datasource, this should not be happening in staging nor production.");
+        DEV_USERS.stream()
+                .flatMap(this::persistDevUserData)
+                .forEach(email -> log.info("Created {} user for development purposes",email));
+        log.info("Added general LiTTiL users for development purposes. You can login via email-addresses and the default password to the dev tenant.");
+    }
 
-        User user = new User();
-        String userId = "auth0|62fd3224f76949850e9eb264";
-        Map<String, Object> appMetadata = Map.of(userIdClaimName, "e87cda48-9a98-41a7-89bb-9ee62441c84c",
-                authorizationsClaimName, Map.of(AuthorizationType.SCHOOL.getTokenValue(), emptyList(), AuthorizationType.GUEST_TEACHER.getTokenValue(), emptyList()));
-        user.setAppMetadata(appMetadata);
-        managementAPI.users().update(userId, user).execute();
+    @Transactional(Transactional.TxType.REQUIRES_NEW)
```

Comment:
ik twijfel of deze Transactional goed wordt toegepast (proxy-class enzo), ik zou het graag per-user doen zodat de rest wel "lukt" als er 1tje faalt